### PR TITLE
Add initial intro to backlogged blog post

### PR DESCRIPTION
**Pull request by arbrown**

### CODE REVIEW

File: content/posts/backlogged-pinball.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Backlogged pinball was designed to demonstrate specific cloud products and
+benefits in a fun, interactive way.
+
+{{< figure src="/images/backlogged-pinball/backlogged-arch.png" width="600px" title="The pinball machine sends pubsub events to the cloud which are processed by Cloud Run and displayed on an interactive web app. Users can deploy their own Cloud Run service to process events and send pubsub messages back to the machine." >}}
+
+In its current iteration, it highlights:
+
+1.  Cloud PubSub: Two-way PubSub communication with a novel source of events
+    (The pinball machine)
+1.  Cloud Run / Cloud functions: Both serverless platforms process pinball
```
Comment (iennae) [inline content/posts/backlogged-pinball.md:28]: Cloud Run functions now.

File: content/posts/backlogged-pinball.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+services.
+
+
+{{< figure src="/images/backlogged-pinball/pinball_suit.jpeg" width="600px" title="A conference attendee playing the Backlogged pinball demo" >}}
+
+## Demo overview
+
+
+
+Backlogged pinball was designed to demonstrate specific cloud products and
```
Comment (NimJay) [inline content/posts/backlogged-pinball.md:19]: **Question:** Just double checking that you meant "cloud" (generic reference) — not "Cloud" (Google Cloud), right?

File: content/posts/backlogged-pinball.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+benefits in a fun, interactive way.
+
+{{< figure src="/images/backlogged-pinball/backlogged-arch.png" width="600px" title="The pinball machine sends pubsub events to the cloud which are processed by Cloud Run and displayed on an interactive web app. Users can deploy their own Cloud Run service to process events and send pubsub messages back to the machine." >}}
+
+In its current iteration, it highlights:
+
+1.  Cloud PubSub: Two-way PubSub communication with a novel source of events
```
Comment (NimJay) [inline content/posts/backlogged-pinball.md:26]: Nitpick: (not a blocker) Consider updating PubSub instances to Pub/Sub.
```suggestion
1.  Cloud Pub/Sub: Two-way Pub/Sub communication with a novel source of events
```

File: content/posts/backlogged-pinball.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+The actual physical pinball machine is a
+[Multimorphic P3](https://www.multimorphic.com/p3-pinball-platform/). In short,
+this is a modular, programmable pinball machine with 2 large HD displays built
+in.
+
+Multimorphic also distributes
+[an SDK](https://www.multimorphic.com/p3-pinball-platform/3rd-party-developers/)
+to facilitate creating new pinball games.
+
+Thus, Backlogged is an application that uses the Unity game engine and the
```
Comment (iennae) [inline content/posts/backlogged-pinball.md:53]: first time you mention Unity game engine so Thus feels jarring, made me question whether I missed something earlier.

File: content/posts/backlogged-pinball.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+Multimorphic SDK to handle interfacing with the hardware (including displays).
+
+## What's next?
+{{< figure src="/images/backlogged-pinball/pinball_interview.jpeg" width="600px" title="Drew showing off the backlogged demo" >}}
+
+The pinball machine currently resides in a Google office in Salt Lake City.  Whenever I am able to, I plan on adding new features/functionality to the demo (with a major focus on cloud code).
+
+I am currently working on a game advisor that uses [llama3.1-70b](https://ollama.com/library/llama3.1:70b) to grade your game and recommend more optimal strategies.
```
Comment (iennae) [inline content/posts/backlogged-pinball.md:61]: is this based on the game or on pinball itself? like does it give recommendations on how to be better at pinball? regardless this is fascinating and makes me want to see more :)

**Review by NimJay — APPROVED** [review]
Looks good to me. :) Approved. No blockers. Good to merge, as-is.

File: content/posts/backlogged-pinball.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+In the future, I'd love to add more features like:
+
+*  Using AI to play the game and optimize for a high score
+*  Connecting two machines to the cloud to demonstrate more interactive
+    scenarios
+    *   This scenario could include two machines in different locations to
+        highlight network latency and resiliency
+*  More interactive features beyond simple messages and emoji.
+
+Any ideas for new features?  I'd love to hear them!
```
Comment (iennae) [inline content/posts/backlogged-pinball.md:74]: how do folks respond to you? (not sure if there is a contact me somewhere)

File: content/posts/backlogged-pinball.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+
+## The code
+
+All backend code (web app, cloud functions, sample reaction services, etc...) is
+in
+[this GitHub repository](https://github.com/GoogleCloudPlatform/backlogged-pinball-backend/).
+
+The pinball game code is heavily dependent on the Multimorphic SDK and sample
```
Comment (iennae) [inline content/posts/backlogged-pinball.md:83]: This is super helpful to see this explicitly. great for sharing this.

**Review by iennae — APPROVED** [review]
Love it!

File: content/posts/backlogged-pinball.md (hunk 1 above)
Comment (arbrown) [reply content/posts/backlogged-pinball.md]: Done

File: content/posts/backlogged-pinball.md (hunk 2 above)
Comment (arbrown) [reply content/posts/backlogged-pinball.md]: Good point! I've updated references to 'Cloud' where appropriate.

File: content/posts/backlogged-pinball.md (hunk 3 above)
Comment (arbrown) [reply content/posts/backlogged-pinball.md]: Done.

File: content/posts/backlogged-pinball.md (hunk 5 above)
Comment (arbrown) [reply content/posts/backlogged-pinball.md:61]: It gives some generic pinball tips (pretty decent ones too) and it's getting better at giving game-specific advice.  Stay tuned for further updates closer to Kubecon!

File: content/posts/backlogged-pinball.md (hunk 6 above)
Comment (arbrown) [reply content/posts/backlogged-pinball.md:74]: I don't have a good way yet... It's on my todo list. I don't want to expose my work (or really personal) email address, so I'm probably going to write some sort of feedback form that forwards emails to both.  And then document that as a blog post 😂

For now, it's just a treasure hunt for highly motivated readers to find my GitHub or LinkedIn accounts!

File: content/posts/backlogged-pinball.md (hunk 4 above)
Comment (arbrown) [reply content/posts/backlogged-pinball.md:53]: Good point!  I added an earlier note clarifying it, though I'd love to write more on this subject at some point.

**Pull request merged**